src/metkit/mars/DHSProtocol.cc

```diff
@@ -370,7 +370,7 @@ Length DHSProtocol::retrieve(const MarsRequest& request)
 {
     Endpoint callbackEndpoint = callback_->endpoint();
 
-    Log::debug() << "DHSProtocol: call back on " << callbackEndpoint << std::endl;
+    LOG_DEBUG_LIB(LibMetkit) << "DHSProtocol: call back on " << callbackEndpoint << std::endl;
```

Comment:
this slightly changes the log behaviour: it now depends on the env variable METKIT_DEBUG instead of DEBUG.
It might be even better...

Reply:
it is more efficient

Reply:
and correct